superset/thumbnails/digest.py

```diff
@@ -18,15 +18,18 @@
 from __future__ import annotations
 
 import logging
-from typing import TYPE_CHECKING
+from typing import Optional, TYPE_CHECKING
```

Comment:
For new code it'd be nice if we always use `from __future__ import annotations` and then use `| None` notation, rather than `Optional[]`